### DISCUSSION TITLE
Hugging face to Android Studio Flutter

### DISCUSSION THREAD

**Discussion by Hereral** [Mobile-Artificial-Intelligence/maid#409] [Q&A]
Hi for our capstone Im building a mobile chatbot with a hugging face model. Is it possible to communicate with hugging face Inference API or endpoints on android studio (flutter). Sorry for asking you this because I found similarities on what Im trying to build.

Sorry this is the only way that I can communicate with you by the way nice project keep it up!

**Comment by danemadsen** [comment]
There is but i haven't implemented it (yet), and Its not really my area so i cant really tell you more sorry.

I know davidmigloz is atleast looking into it for langchain.dart (which maid depends on) so maybe ask him if you would like more detail

https://github.com/davidmigloz/langchain_dart/tree/main/packages/langchain_huggingface

**Reply by Hereral** [reply]
Thanks i'll take a look into it
